docs/source/cloud-setup/cloud-permissions/nebius.rst

```diff
@@ -9,30 +9,19 @@ Service account
 To use *Service Account* authentication, follow these steps:
 
 1. **Create a Service Account** using the Nebius web console.
-2. **Generate PEM Keys**:
+2. **Generate Service Account Credentials**:
 
-.. code-block:: shell
-
-   openssl genrsa -out private.pem 4096
-   openssl rsa -in private.pem -outform PEM -pubout -out public.pem
+Run the following command to generate the service account credentials:
 
-3.  **Generate and Save the Credentials File**:
-
-* Save the file as `~/.nebius/credentials.json`.
-* Ensure the file matches the expected format below:
+.. code-block:: shell
 
-.. code-block:: json
+   nebius iam auth-public-key generate \
+   --service-account-id $SA_ID \
+   --output ~/.nebius/credentials.json
```

Comment:
Curious, this won't require local credentials to be setup yet, right? I suppose it will jump to a web browser for authentication?